frontend/src/components/header/header.tsx

```diff
@@ -26,7 +26,18 @@ import { PrivateElements } from './private-elements/private-elements';
 import styles from './styles.module.scss';
 
 export const Header = (): React.ReactElement => {
-  const [value, setValue] = useState(0);
+  let tabNumber;
+  switch (window.location.pathname) {
+    case AppRoute.SEARCH:
+      tabNumber = 0;
+      break;
+    case AppRoute.ABOUT:
+      tabNumber = 1;
+      break;
+    default:
+      tabNumber = null;
+  }
```

Comment:
Please, don't use the number to define the active tab element. Because it'll complicate changing the order of the tabs. Store unique identified of the tab - it's name

Reply:
@oleksandr-danylchenko but mui Tabs component receives only the number of the chosen tab

Reply:
@ashonuyak 
Then you need to use the `enum`. It's a perfect tool if you need to attach a human-readable name to some unknown numeric value in a predictable manner